internal/jspackages/js_yarn.go

```diff
@@ -24,6 +24,17 @@ func FindYarnDependencies(wd string) (depbot.Dependencies, error) {
 
 	dependencies := depbot.Dependencies{}
 
+	depRegex, err := regexp.Compile(`(?:"?([^\s]+?)@)`)
```

Comment:
@bcuadrad1 notice this change. If these regexp do not compile we should just return. No need to get into the parsing.

Reply:
Also, notice that the names of the variables get more specific given we're now defining them with a larger scope.